db-engine/src/main/java/com/splicemachine/db/iapi/types/SQLTimestamp.java

```diff
@@ -1584,6 +1586,49 @@ public void write(UnsafeRowWriter unsafeRowWriter, int ordinal) {
 				holder.cursor = 12;
 			}
 	    }
+
+	/**
+	 *
+	 * Write Element into Positioned Array
+	 *
+	 * @param unsafeArrayWriter
+	 * @param ordinal
+	 * @throws StandardException
+     */
+	@Override
+	public void writeArray(UnsafeArrayWriter unsafeArrayWriter, int ordinal) throws StandardException {
+		if (isNull())
+			unsafeArrayWriter.setNull(ordinal);
+		else {
+			byte[] data = new byte[12];
```

Comment:
Timestamp is serialized as 3 ints (4 bytes).  Are you asking me to put the magic 12 in a static variable somewhere?